views/js/pciCreator/dev/textReaderInteraction/pciCreator.json

```diff
@@ -0,0 +1,27 @@
+{
+    "typeIdentifier": "textReaderInteraction",
+    "label": "Text reader",
+    "description": "The Paging widget combines a scrolling widget with additional paging controls.",
+    "version": "0.1",
+    "author": "Aleh Hutnikau",
+    "email": "hutnikau@1pt.com",
+    "libraries": [
+        "IMSGlobal/jquery_2_1_1",
+        "./runtime/js/renderer",
+        "./runtime/js/tabs"
+    ],
+    "css": [
+        "./runtime/css/textReaderInteraction.css" 
+    ],
+    "entryPoint": "./runtime/textReaderInteraction.amd.js",
```

Comment:
note : just to let you know for the future: the amd in textReaderInteraction.amd.js is not actually need. I added "amd" for likertScaleInteraction because they were two candidate implementation for PCI, one with and one without. And I leave the successful implementation in place. You may leave that this time but for your next pci, you can simply omit that ;)